src/main/java/com/rtb/projectmanagementtool/task/TaskServlet.java

```diff
@@ -74,18 +90,36 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       }
     }
 
-    // // Get Comments
+    // Get Comments
     // int quantity = Integer.parseInt(request.getParameter("quantity"));
     // String sortBy = request.getParameter("sortBy");
     // String sortDirection = request.getParameter("sortDirection");
+    CommentController commentController = new CommentController(datastore);
+    ArrayList<CommentData> comments = new ArrayList<>();
+    HashMap<CommentData, String> commentsMap = new HashMap<>();
```

Comment:
I would advice creating a new object to hold this aggregation, instead of piggybacking off of the map's key/value pair.  Maybe something like `CommentDisplayData`.  This would be useful in the future as well if you need to add more  derived info in there.